core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -357,4 +355,15 @@ public String changePrimaryGateway(Stack stack) throws CloudbreakException {
             throw new CloudbreakException("Primary gateway change is not possible because there is no available node for the action");
         }
     }
+
+    private void decoratePillarWithJdbcConnectors(Cluster cluster, Map<String, SaltPillarProperties> servicePillar) {
+        Set<RDSConfig> rdsConfigs = rdsConfigService.findByClusterId(cluster.getOwner(), cluster.getAccount(), cluster.getId());
+        Map<String, Object> connectorJarUrlsByVendor = rdsConfigs.stream()
+                .filter(rds -> StringUtils.isNoneEmpty(rds.getConnectorJarUrl()))
+                .collect(Collectors.toMap(RDSConfig::getDatabaseEngine, RDSConfig::getConnectorJarUrl));
+        Map<String, Object> jdbcConnectors = Collections.singletonMap("jdbc_connectors", connectorJarUrlsByVendor);
```

Comment:
`jdbcConnector` not need for the if statement, so possible to initialize only inside the if